src/OrleansCounterControl/Microsoft.Orleans.CounterControl.targets

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="15.0" DefaultTargets="Build">
```

Comment:
Talked in person. Until we ship that, let's at least remove which ToolsVersion is used, so that it implicitly uses the one where the solution is running